### ISSUE TITLE
LAION-Pose Link

### ISSUE THREAD

**Issue by jfischoff** [mayuelala/FollowYourPose#12]
Hi, 

I was looking for the links the LAION-Pose dataset from the paper. I was looking to help a team for the Huggingface JAX sprint train a MediaPipe hand tracking annotator for ControlNet. 

Is the dataset publicly available?

**Comment by mayuelala** [comment]
Hi
thank you for your attention, the dataset is clearing and will be published as soon as possible.

**Comment by juxuan27** [comment]
Hi, @jfischoff. We have provide a dataset very similar to LAION-Pose (with pose estimator that trained on [Human-Art](https://github.com/IDEA-Research/HumanArt) and thus provide multi-scenario poses, e.g. oil painting, sculpture). You can find the dataset [here](https://github.com/IDEA-Research/HumanSD) and see if it is helpful.

**Comment by jfischoff** [comment]
@juxuan27 thanks for the link! 

Also, I checked out your project. I hadn't seen it before but very cool work. The heatmap concept is very interesting. 

I have been training a controlnet that was previous frame + pose and we really struggled to get the pose to be used properly. We ultimately came to the conclusion that we needed to modify the loss function to highlight the pose areas. I'll have to checkout the paper in more detail, but I'm very interested in those type of results.
